src/app.ts

```diff
@@ -71,6 +76,64 @@ const main = async () => {
     res.json({ success: 'true' });
   });
 
+  const storage = multer.memoryStorage();
+  const upload = multer({ storage: storage });
+
+  // Not really sure where in the code base go, but here's the working route!
+  // I did my part :)
+  // Is just using `/flyers/` the right route here?
+  app.post('/flyers/', upload.single('file'), async (req, res) => {
+    // Ensure there is an image file present
+    if (!req.file) {
+      return res.status(400).json({ error: 'No file uploaded' });
+    }
+    const {
+      categorySlug,
+      endDate,
+      flyerURL,
+      location,
+      organizationID,
+      startDate,
+      title,
+    } = req.body;
+
+    // Is there a better way to do this? (assert request body has required fields)
+    if (
+      categorySlug == null ||
+      endDate == null ||
+      flyerURL == null ||
+      location == null ||
+      organizationID == null ||
+      startDate == null ||
+      title == null
+    ) {
+      return res.status(400).json({ error: 'Missing a required field' });
+    }
```

Comment:
You can store the required fields as strings in an array and use `.every()` on the array to check if the value is contained in the object's keys. An example of this is [here](https://stackoverflow.com/questions/55871220/how-to-check-if-a-json-doc-has-all-required-keys-in-node-js).

I honestly think what you have is fine, though, it's probably more readable than the other approach